nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -253,7 +253,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
             <WinMDFullPath Include="@(_CppWinRTProjectWinMDItems->FullPath()->Distinct()->ClearMetadata())" Condition="'$(CppWinRTGenerateWindowsMetadata)' == 'true'">
                 <TargetPath>$([System.IO.Path]::GetFileName('$(CppWinRTProjectWinMD)'))</TargetPath>
                 <Primary>true</Primary>
-                <Implementation Condition="'$(TargetExt)' == '.dll'">$(WinMDImplementationPath)$(TargetName)$(TargetExt)</Implementation>
+                <Implementation Condition="'$(TargetExt)' == '.dll' or '$(TargetExt)' == '.exe'">$(WinMDImplementationPath)$(TargetName)$(TargetExt)</Implementation>
```

Comment:
VS never supported this for CX and so when support for WinRT was added we added the same limitation. We could possibly remove the TargetExt check alltogether. Not sure if you would run into any other roadblocks with that.

Reply:
local servers are commonly implemented as a .exe (Win32 service is the other option).

Reply:
@jlaanstra and @kennykerr do you have advice on doing an OS and terminal build? Given this did not work before perhaps this is safe enough to not require that (me trying to shirk this work), let me know if you think so.

Reply:
😉 I do like the fact that it is a small and targeted change. We've just had a lot of problems with seemingly innocuous changes to the msbuild support that have had surprising compatibility issues in different environments. @jlaanstra has done this validation a few times - he may have some tips to quickly smoke out any issues.

Reply:
@ChrisGuzak the way I generally go about this is to replace the CppWinRT props/targets used by a project and then do a full build.  For the OS this means replacing the files under tools\cppwinrt (I can share a script I use to build a few known WinRT directories) and for terminal it means replacing the files in the expanded NuGet package under the packages folder and building terminal per instructions in their repo.

Reply:
Terminal has been validated. working on os.2020 next.